fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -156,7 +156,7 @@ public void contextInitialized(ServletContextEvent event) {
 
                 // Gather up the NATS TLS properties.
                 Properties tlsProps = new Properties();
-                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED).toString());
+                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED, true).toString());
```

Comment:
```suggestion
                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED, Boolean.TRUE).toString());
```

Reply:
pattern below is Boolean.True

Reply:
meh, can i change the below ones to "true" instead?

Reply:
you'd have to change a bunch of spots in this class. I think this is consistent